Libraries/Components/SegmentedControlIOS/__tests__/SegmentedContolIOS-test.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
Amazing! Thanks for writing these tests!

Reply:
You're welcome! Thanks for your reviews and your patience with me!